docs/src/pages/[platform]/connected-components/liveness/troubleshooting/index.page.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: Face Liveness Troubleshooting
+description: Troubleshoot your issues with the Amplify UI FaceLivenessDetector component.
+supportedFrameworks: react|android|swift
+---
+
+import { Fragment } from '@/components/Fragment';
+
+## Troubleshooting
```

Comment:
Sorry I didn't suggest this originally, didn't notice it should just be removed till I saw it live
```suggestion
```